moninag/static/src/ts/app/navs/side-nav/side-nav.component.ts

```diff
@@ -6,15 +6,59 @@ import { Service } from '../../services/services';
 import { Check } from '../../checks/check';
 import { SideNavService } from './side-nav.service';
 
+@Component({
+    selector: 'ngb-accordion',
+    template: `<ng-content></ng-content>`
+})
+
+export class NgbAccordion {
+    private groups: Array<NgbAccordionGroup> = [];
+}
+
+@Component({
+    selector: 'ngb-accordion-group', 
+    inputs: ['heading', 'isOpen', 'isDisabled'],
+    template: `
+        <div>
+            <div class="isDisabled" (click)="toggleOpen($event)">{{heading}}</div>
+            <div [hidden]="!isOpen">
+                <div>
+                    <ng-content></ng-content>
+                </div>
+            </div>
+        </div>
+  `
+})
+
+export class NgbAccordionGroup {
+
+    private isDisabled: boolean;
+    private _isOpen: boolean = false;
```

Comment:
according to Microsoft/TypeScript coding guidelines: "Do not use "_" as a prefix for private properties."
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines